From f2e41ec4125518a7643fe14ec74277325abbab64 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Thu, 23 Jan 2020 08:31:46 -0600
Subject: [PATCH 02/12] Add copying mkimage_uboot to NXP's version of mkimage.

The script is expecting to be able to use the mkimage from U-Boot.
Without this copy, the script fails to build.  I don't know if it
is necessary for Yocto, but I doubt that it is.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 txfrToMkImg-N.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/txfrToMkImg-N.sh b/txfrToMkImg-N.sh
index b2a8228e80..f18af6686f 100755
--- a/txfrToMkImg-N.sh
+++ b/txfrToMkImg-N.sh
@@ -1,6 +1,7 @@
 #!/bin/bash
 #
 cp ./arch/arm/dts/beacon-imx8mn-kit.dtb ../imx-mkimage/iMX8M/fsl-imx8mn-evk.dtb
+cp ./tools/mkimage ../imx-mkimage/iMX8M/mkimage_uboot
 cp ./u-boot-nodtb.bin ../imx-mkimage/iMX8M
 cp ./u-boot-dtb.bin ../imx-mkimage/iMX8M
 cp ./spl/u-boot-spl.bin ../imx-mkimage/iMX8M
-- 
2.17.1

